devices/flash-nrf9160/flashdrv.c

```diff
@@ -0,0 +1,154 @@
+/*
+ * Phoenix-RTOS
+ *
+ * plo - operating system loader
+ *
+ * NRF9160 Flash driver
+ *
+ * Copyright 2020 Phoenix Systems
+ * Author: Aleksander Kaminski
```

Comment:
I guess it's based on my code, but you should add yourself to authors

Reply:
I wasn't sure, because in this driver I didn't add my code, just adjusted the existing one